HW_pandas/HW_pandas.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import seaborn as sns
+from sklearn.preprocessing import StandardScaler
+
+url = "https://raw.githubusercontent.com/Serfentum/bf_course/master/14.pandas/train.csv"
+train_df = pd.read_csv(url)
+
+cols = ['pos', 'A', 'C', 'T', 'G']
+nucleotides = train_df[cols].groupby('pos').sum()
+nucleotides.plot.bar(stacked=True, xlabel='pos', ylabel='number of reads', rot=50, figsize=(40, 10),
+                     fontsize=30)
+
+cols = ['pos', 'reads_all', 'mismatches', 'deletions', 'insertions']
+train_part = train_df.query('matches > matches.mean()')[cols]
+train_part.to_csv('Data/train_part.csv')
```

Comment:
Пути к файлам следуеи указывать через `os.path.join`
```suggestion
train_part.to_csv(os.path.join('Data', 'train_part.csv'))
```